### PR TITLE
Add a CONTRIBUTING.md

### LINKED ISSUE #998: Add a CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by rgrunber**
- Fixes #998

Signed-off-by: Roland Grunberg <rgrunber@redhat.com>

Let me know if I've missed anything. Can always be improved as we go.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
+# How to Contribute
+
+Contributions are essential for keeping this language server great. We try to keep it as easy as possible to contribute changes and we are open to suggestions for making it even easier. There are only a few guidelines that we need contributors to follow.
+
+## Development
+
+### Installation Prerequisites:
+
+  * [JDK 8+](http://www.oracle.com/technetwork/java/javase/downloads/index.html)
```
Comment (datho7561) [inline CONTRIBUTING.md:9]: Maybe link to AdoptOpenJDK instead? It looks like it's [joining the Eclipse foundation](https://blog.adoptopenjdk.net/2020/06/adoptopenjdk-to-join-the-eclipse-foundation/)

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+	```bash
+	$ ./mvnw verify
+	```
+	or for Windows:
+	```bash
+	$ mvnw.cmd verify
+	```
+
+### Debug
+
```
Comment (datho7561) [inline CONTRIBUTING.md:25]: I think you should explicitly mention that you need to connect to the language server through remote debugging.

**Review by datho7561 — CHANGES_REQUESTED** [review]
I think the debug section needs a bit of work, but otherwise looks good.

File: CONTRIBUTING.md (hunk 1 above)
Comment (rgrunber) [reply CONTRIBUTING.md]: Done. We may also want to change this on the vscode-xml page.

File: CONTRIBUTING.md (hunk 2 above)
Comment (rgrunber) [reply CONTRIBUTING.md:25]: Done.

**Review by datho7561 — APPROVED** [review]

**Pull request merged**